pkg/cluster/info.go

```diff
@@ -59,6 +60,15 @@ func NewInfo(clusterName string, config *rest.Config) (*Info, error) {
 		return nil, errors.Wrap(err, "error creating client producer")
 	}
 
+	brokers := &v1alpha1.BrokerList{}
+	err = info.ClientProducer.ForGeneral().List(context.TODO(), brokers, controllerClient.InNamespace(metav1.NamespaceAll))
+
+	if err == nil {
+		info.Broker = brokers.Items
+	} else if !apierrors.IsNotFound(err) && !meta.IsNoMatchError(err) {
+		return nil, errors.Wrapf(err, "error retrieving Broker")
+	}
```

Comment:
Not one. This is used by `recover-brokerinfo` extensively ([first time](https://github.com/submariner-io/subctl/pull/500/files#diff-ae7e6105a69a92b24a4c33457e32203a26eab3dd52a5fc97b6e5a98c7e41d5eeR48), [second time](https://github.com/submariner-io/subctl/pull/500/files#diff-ae7e6105a69a92b24a4c33457e32203a26eab3dd52a5fc97b6e5a98c7e41d5eeR80)). This make `recover-brokerinfo` logic easier. It can also be used by `uninstall` simplifying [findBrokerNamespace](https://github.com/submariner-io/subctl/blob/devel/pkg/uninstall/all.go#L324)

Reply:
yes but in those places the brokers can be still be retrieved on-demand via a function  and stored locally, ie they don't need to be retrieved up front and stored globally in the `Info` object.